packages/pigeon/run_tests.sh

```diff
@@ -67,6 +67,22 @@ test_pigeon_android() {
   rm -rf $temp_dir
 }
 
+# test_null_safe_dart(<path to pigeon file>)
+#
+# Compiles the pigeon file to a temp directory and attempts to run the dart
+# analyzer on it with null safety turned on.
+test_null_safe_dart() {
+  temp_dir=$(mktemp -d -t pigeon)
+
+  pub run pigeon \
+    --input $1 \
+    --dart_null_safety \
+    --dart_out $temp_dir/pigeon.dart
+
+  dartanalyzer $temp_dir/pigeon.dart --fatal-infos --fatal-warnings --packages ./e2e_tests/test_objc/.packages --enable-experiment=non-nullable
```

Comment:
what about also calling `dart` with `--enable-experiment=non-nullable` to verify that the code can compile?

Reply:
`dartanalyzer` does check that it can compile

Reply:
I see